test/cppsuite/src/main/thread_worker.cpp

```diff
@@ -316,4 +291,96 @@ thread_worker::get_assigned_collection_count() const
     uint64_t collection_count = db.get_collection_count();
     return collection_count / thread_count + (collection_count % thread_count > id);
 }
+
+/*
+ * Returns true if a transaction can be committed as determined by the op count and the state of the
+ * transaction.
+ */
+bool
+thread_worker::can_commit()
```

Comment:
shouldn't this be `const` too?

Reply:
Actually it can't be because it calls _txn which throws a compile error.